regulations/generator/html_builder.py

```diff
@@ -229,3 +234,19 @@ def process_node(self, node, indexes=None):
         label_id = '-'.join(node['label'])
         node['accepts_comments'] = label_id in self.diff_applier.diff
         node['comments_calledout'] = label_id in self.diff_applier.diff
+
+        node['has_diff'] = label_id in self.diff_applier.diff
```

Comment:
Fair enough. I thought `has_diff` might be needed more widely, but we can cross that bridge if it ever arises.